src/plugins/controls/public/react_controls/controls/types.ts

```diff
@@ -62,7 +62,6 @@ export type ControlApiInitialization<ControlApi extends DefaultControlApi = Defa
     'serializeState' | 'getTypeDisplayName' | 'clearSelections'
   >;
 
-// TODO: Move this to the Control plugin's setup contract
```

Comment:
We decided to **not** allow registering controls via the Control plugin's setup contract, so I removed this `TODO`